WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -39,19 +39,6 @@ struct PointOfSaleDashboardView: View {
         }
         .toolbarBackground(Color.toolbarBackground, for: .bottomBar)
         .toolbarBackground(.visible, for: .bottomBar)
-        .sheet(isPresented: $viewModel.showsCardReaderSheet, content: {
```

Comment:
This needs to be here because now the "Connect now" button in toolbar does not work.

Reply:
now it only works if the totals are presented.

Reply:
Ohh great catch! I will fix this today.

Reply:
I was looking at this now and even if I change to how it was the sheet does not show, did we change something behind the scenes on how we set showsCardReaderSheet?

Reply:
> I was looking at this now and even if I change to how it was the sheet does not show, did we change something behind the scenes on how we set showsCardReaderSheet?

It does work for me with the changes on https://github.com/woocommerce/woocommerce-ios/pull/13226, can you check if you can still reproduce the issue on that branch?